docs/GettingStartedGuide.md

```diff
@@ -50,7 +50,7 @@ HERE OLP SDK consists of specific libraries, each with distinct functionality.
 
 [Volatile Layer Read example](./examples/nodejs-read-volatile-layer.md) shows how to retrieve layer metadata, partition metadata, and partition data from volatile layer using OLP SDK for TypeScript.
 
-## <a name="CatalogClient"></a> CatalogClient
+## <a name="catalog-client"></a> CatalogClient
 
 The main entry point to the OLP is `CatalogClient`. This class provides a high-level interface for retrieval of `OLP` catalog data and configuration, and defines the following operations:
 
```

Comment:
Comment to lines 57-58: add full stops at the end of the list items.

Reply:
Comment to line 60
Change to:
"For more details on how to use the `CatalogClient` object, see the [catalog usage example](./examples/nodejs-examples.md)."